### PR TITLE
Spanish translation ch12-02-reading-a-file.md 1/1

### PR DESCRIPTION
**Pull request by chicooou**
- words 660
- lines 1-112

### CODE REVIEW

File: second-edition/src/ch12-02-reading-a-file.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+## Leyendo un Archivo
 
-Now we’ll add functionality to read the file that is specified in the
-`filename` command line argument. First, we need a sample file to test it with:
-the best kind of file to use to make sure `minigrep` is working is one with a
-small amount of text over multiple lines with some repeated words. Listing 12-3
-has an Emily Dickinson poem that will work well! Create a file called
-*poem.txt* at the root level of your project, and enter the poem “I’m Nobody!
-Who are you?”
+Ahora añadiremos funcionalidad para leer el archivo especificado en el argumento de la
```
Comment (thecodix) [inline second-edition/src/ch12-02-reading-a-file.md]: Please revise that spanish punctuation signs ! need also an opening matching one :)

**Comment by thecodix** [comment]
Small comment, please review when you have some time.

Very good work in general, thanks for contributing to this! 🦀

File: second-edition/src/ch12-02-reading-a-file.md (hunk 1 above)
Comment (chicooou) [reply second-edition/src/ch12-02-reading-a-file.md]: all fixed 👍

**Comment by thecodix** [comment]
machine translated

**Pull request closed without merging**